lucien-ios/LucienConstants.swift

```diff
@@ -15,8 +15,7 @@ final class LucienConstants {
     static let keyboardHeightPadding = CGFloat(50)
 
     // MARK: - Form Constants
-
-    static let textFieldBorderOffset = CGFloat(48)
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)